### DISCUSSION TITLE
DDR cf_tunnel

### DISCUSSION THREAD

**Discussion by FrozenEmpire** [ChilliHugger/The-Lords-Of-Midnight#204] [Code Algorithm Questions] [answered]
Hi Chris,

Very minor point really, but the resetting of cf_tunnel during displacement seems unnecessary to me, since displacement only occurs during battles, and battles can only occur outside of tunnels anyhow ?

https://github.com/ChilliHugger/The-Lords-Of-Midnight/blob/f0242bb21c4e99e8afa61f33cf421807c217fb95/main/midnight/src/tme/scenarios/ddr/ddr_character.cpp#L1399

**Comment by IcemarkUK** [comment, marked as answer]
Yes I'd agree - that's correct
